indexer/data_header.hpp

```diff
@@ -71,6 +72,7 @@ namespace feature
     /// @name Serialization
     //@{
     void Save(FileWriter & w) const;
+    void SaveRegionData(FileWriter & w) const;
```

Comment:
Why not to unify Save and SaveRegionData - these methods are called in sequence, so I prefer to have a single method instead of two.